### PR TITLE
doc: Move the note on sending requests to multiple servers higher.

### PR DESCRIPTION
**Pull request by thorduri**
Requested by the solutions engineers, as this is pretty buried

### CODE REVIEW

File: doc/Getting_Started_Writing_Clients.adoc [hunk 1]
```diff
@@ -73,6 +73,10 @@ The servers that Yubico provides are:
 
 These servers are hosted in different places and by different organizations.
 
+Clients should send authentication requests to all of them in parallel, and
+utilize the first response recieved for added reliability should one or more
```
Comment (dainnilsson) [inline doc/Getting_Started_Writing_Clients.adoc]: A technical detail here is that the first response for a valid request might be a REPLAYED_REQUEST due to network latency etc. I believe awaiting the first non-REPLAYED_REQUEST response is the correct behavior (or until all requests have returned, and then failing as the request truly was replayed).

Comment (thorduri) [inline doc/Getting_Started_Writing_Clients.adoc]: Yes, I left that bit out on purpose.

Might be more confusion come to think of it, how about adding a link to one of our clients as an example instead ?

```
Clients should send authentication requests to all of them in parallel, and
utilize the first response received for added reliability should one or more
of the api servers be offline, see (https://github.com/Yubico/yubico-c-client)[Yubico C client] for technical details.
```

Yes, this is a punt until there is a better validation spec.

Comment (dainnilsson) [inline doc/Getting_Started_Writing_Clients.adoc]: I like that idea. 👍

Comment (thorduri) [inline doc/Getting_Started_Writing_Clients.adoc]: Done.

**Comment by dainnilsson** [comment]
👍

**Pull request merged**